src/redux/configStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+);
```

Comment:
- [ ] Please remember to import `applyMiddleware` and use it inside the `createStore` 😄